core/templates/dev/head/pages/collection_editor/collection_editor_navbar_directive.html

```diff
@@ -55,14 +55,14 @@
     </li>
   </ul>
 
-  <ul class="nav navbar-nav oppia-navbar-nav navbar-right">
+  <ul class="nav navbar-nav oppia-navbar-nav navbar-right" ng-if="!isCollectionLoading()">
```

Comment:
I think it might be the better poison since if we do show it, it might be incorrect.